nni/compression/pytorch/speedup/jit_translate.py

```diff
@@ -482,316 +175,326 @@ def forward(self, obj):
-    elif value.startswith('{'):
-        # TODO Support set values in the future
-        value = set()
-    elif '.' in value:
-        # float value
-        value = float(value)
-    else:
-        # integer value
-        value = int(value)
-    return ConstantModule(value)
-
-def upsample_bilinear2d_python(node, speedup):
-    class UpsampleModule(torch.nn.Module):
-        def __init__(self, size_list, scale_list):
-            super(UpsampleModule, self).__init__()
-            self.size_list = size_list
-            self.scale_list = scale_list
-
-        def forward(self, *args):
-            """
-            The first input of args is the target tensor to upsample
-            , the following parameters is useless, because we already
-            get the size_list and the scale_list by parsing the cpp_nodes.
-            """
-            return torch.nn.functional.upsample_bilinear(args[0],
-                                                         size=self.size_list, scale_factor=self.scale_list)
-    c_node = node.key_node
-    inputs = list(c_node.inputs())
-    size_list_node = inputs[1].node()
-    scale_list_node = inputs[3].node()
-    size_list = None
-    scale_list = None
-
-    if size_list_node.kind() == 'prim::ListConstruct':
-        size_list = translate_list(inputs[1], speedup)
-    if scale_list_node.kind() == 'prim::ListConstruct':
-        scale_list = translate_list(inputs[3], speedup)
-    return UpsampleModule(size_list, scale_list)
-
-
-def upsample_nearest2d_python(node, speedup):
-    class UpsampleModule(torch.nn.Module):
-        def __init__(self, size_list, scale_list):
-            super(UpsampleModule, self).__init__()
-            self.size_list = size_list
-            self.scale_list = scale_list
-
-        def forward(self, *args):
-            """
-            The first input of args is the target tensor to upsample
-            , the following parameters is useless, because we already
-            get the size_list and the scale_list by parsing the cpp_nodes.
-            """
-            return torch.nn.functional.upsample_nearest(args[0],
-                                                        size=self.size_list, scale_factor=self.scale_list)
-    c_node = node.key_node
-    inputs = list(c_node.inputs())
-    size_list_node = inputs[1].node()
-    scale_list_node = inputs[2].node()
-    size_list = None
-    scale_list = None
 
-    if size_list_node.kind() == 'prim::ListConstruct':
-        size_list = translate_list(inputs[1], speedup)
-    if scale_list_node.kind() == 'prim::ListConstruct':
-        scale_list = translate_list(inputs[2], speedup)
-    return UpsampleModule(size_list, scale_list)
+    def __init__(self, func: Callable, positional: List[Any], keyword: Dict[str, Any],
+                undetermined: List[Union[int, str]], special_treat: Dict[Union[int, str], Callable]):
+        if not callable(func):
+            raise TypeError('the \'func\' argument must be callable')
```

Comment:
It's more clear to write `'the "func" argument must be callable'`

Reply:
Are all the "\'" in output infos should be changed to "